File: webservices/src/main/java/fieg/modulos/cadastro/produtodadoscontabil/dto/ProdutoDadoContabilDTO.java
package fieg.modulos.cadastro.produtodadoscontabil.dto;

import fieg.modulos.cadastro.dadocontabil.dto.DadoContabilDTO;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
public class ProdutoDadoContabilDTO {



    private Integer idProdutoDadoContabil;
    private DadoContabilDTO dadoContabil;
    private Integer idProduto;
    private String produto;
    private String status;
    private String codProdutoProtheus;
    private BigDecimal preco;
    private Integer idSistema;
    private String dmed;
    private Integer idOperadorInclusao;
    private LocalDateTime dataInclusao;
    private Integer idOperadorAlteracao;
    private LocalDateTime dataAlteracao;
    private LocalDateTime dataInativacao;

}

File: web/src/models/DTOs/Produtos/ProdutoDadoContabilDTO.ts
import { DadoContabilDTO } from "../Contabil/DadoContabilDTO";

export interface ProdutoDadoContabilDTO {
  dadoContabil: DadoContabilDTO;
  idProdutoDadoContabil: number;
  produto: string;
  status: string;
  codProdutoProtheus: string;
  dataInativacao: number | null;
  preco: number;
  idProduto: number;
  idSistema: number;
  dmed: string;
}
